csv_functions.py

```diff
@@ -693,11 +693,11 @@ def taspr_csv(data, endpoint):
             filename_data_name = "Precipitation"
 
     elif endpoint == "proj_precip":
-        coords = ["return_interval", "duration", "model", "era"]
+        coords = ["exceedance_probability", "duration", "model", "era"]
         values = ["pf", "pf_lower", "pf_upper"]
         fieldnames = coords + values
         csv_dicts = build_csv_dicts(data, fieldnames, values=values)
-        metadata = "# return_interval is used for the Annual Exceedance Probability (1 / return_interval)\n"
+        metadata = "# exceedance_probability is the annual exceedance probability\n"
```

Comment:
I think we should add the unit of "percent" to the exceedance probability value for clarity. 
```suggestion
        metadata = "# exceedance_probability is the annual exceedance probability in percent\n"
```